app/models/credit.rb

```diff
@@ -36,16 +36,28 @@ def self.add_to(user_or_org, amount)
     update_cache_columns(user_or_org)
   end
 
+  # Remove an :amount of credits from the given :user_or_org.
+  #
+  # @param user_or_org [#credits] the "owner" of the credits.
+  # @param amount [Integer] the amount of credits to remove from the
+  #
+  # @note If you specify removing more credits than the owner has, it
+  #       will remove all of their credits but not create a negative
+  #       balance.
+  #        owner.
```

Comment:
this last comment line looks like it might be leftover from an edit?

Reply:
Good catch…silly text editor moving lines.